open-craft/opencraft
# Add periodic task to delete old log entries

This adds a new task that tries to efficiently delete old log entries.

**Testing instructions**:

1. Put branch on stage.
2. Run the job manually in the shell:

```
In [1]: from instance import tasks
In [2]: tasks.delete_old_logs()
```

3. Wait till done.
4. Go check any logs created older than 60 days and ensure they no longer exist.
